internal/store/review_test.go

```
@@ -0,0 +1,174 @@
+	if err := st.SetReviewState("session", "/repo", want); err != nil {
+		t.Fatal(err)
+	}
+	if err := st.Close(); err != nil {
+		t.Fatal(err)
+	}
+
+	st, err = Open(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer st.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `st.Close` is not checked

(errcheck)